fleetspeak/src/admin/cli/cli.go

```diff
@@ -47,7 +47,7 @@ func Usage() {
 	n := path.Base(os.Args[0])
 	fmt.Fprintf(os.Stderr,
 		"Usage:\n"+
-			"    %s listclients\n"+
+			"    %s listclients [[client_id]]\n"+
```

Comment:
This format looks a bit unfamiliar to me. Totally up to you, but how about using ellipsis to indicate that you expect a list of client ids (as separate args):

```
listclients <client_id> <client_id>...
```

See http://docopt.org/